### PR TITLE
feat(query): Added Request Timeout Not Properly Set for Kubernetes

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
- Added Request Timeout Not Properly Set for Kubernetes

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by joaorufi — CHANGES_REQUESTED** [review]
Please, remove payload.json file

File: assets/queries/k8s/request_timeout_not_properly_set/query.rego [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--request-timeout flag should not be set to more than 300 seconds",
+		"keyActualValue": "--request-timeout flag has an invalid value",
+		"searchLine": common_lib.build_search_line(split(specInfo.path, "."), [types[x], j, "command"]),
+	}
+}
```
Comment (rafaela-soares) [inline assets/queries/k8s/request_timeout_not_properly_set/query.rego:44]: Is this policy really necessary?

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/request_timeout_not_properly_set/query.rego [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	getSeconds(flag_value)> value
+} else {
+	args := container.args
+	startswith(args[a], flag)
+	flag_value := split(args[a], "=")[1]
+	getSeconds(flag_value)> value
+}
+
+getSeconds(time)=seconds{
+	regex.match("^(\\d{1,3}[h])$", time)
```
Comment (rafaela-soares) [inline assets/queries/k8s/request_timeout_not_properly_set/query.rego:39]: Maybe we should not limit the number of digits

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — DISMISSED** [review]
LGTM 🚀

**Review by joaorufi — APPROVED** [review]

**Pull request merged**